bottomsheet-commons/src/main/java/android/support/v4/app/AccessFragmentInternals.java

```diff
@@ -0,0 +1,10 @@
+package android.support.v4.app;
+
+public class AccessFragmentInternals {
+    private AccessFragmentInternals() {
+    }
+
+    public static int getContainerId(Fragment fragment) {
+        return fragment.mContainerId;
+    }
+}
```

Comment:
Not sure the best way to handle this. DialogFragment makes use of this field that's not part of the public api to determine if it's showing in a dialog or normally. Other possible solutions are to use reflection or find some other way to obtain the same information without that field.